src/renderer/src/routes/Onboarding/PrivacyPolicyScreen.tsx

```diff
@@ -21,18 +21,22 @@ const ContentBox = styled('div')({
 	width: '55%',
 	height: '80%',
 	backgroundColor: '#FFFFFF',
-	padding: '20px 30px',
+	padding: 20,
 	borderRadius: 8,
 	border: `1px solid ${BLUE_GREY}`,
-	margin: '16px auto',
+	margin: '12px auto',
 	boxShadow: '0px 4px 4px 0px rgba(0, 0, 0, 0.02)',
 	overflowY: 'auto',
 })
 
 export function PrivacyPolicyScreen() {
+	const navigate = useNavigate()
 	return (
 		<Container>
-			<OnboardingTopMenu currentStep={1} />
+			<OnboardingTopMenu
+				currentStep={1}
+				onBackPress={() => navigate({ to: '/Onboarding/DataPrivacy' })}
+			/>
 			<ContentBox>
 				<PrivacyPolicy />
```

Comment:
Why is the privacy policy a nested component. It basically a container inside a container. it just seems like an unnecessary nesting

Reply:
I did that in the same way I did it on mobile because the Privacy Policy is used in a separate place within the settings and is almost exactly the same so that way it can be reused as a component there.